eng/Version.Details.xml

```diff
@@ -1,59 +1,59 @@
     </Dependency>
-    <Dependency Name="NETStandard.Library" Version="2.1.0-prerelease.19158.1">
+    <Dependency Name="NETStandard.Library" Version="2.1.0-prerelease.19222.1">
       <Uri>https://github.com/dotnet/standard</Uri>
-      <Sha>8ef5ada20b5343b0cb9e7fd577341426dab76cd8</Sha>
+      <Sha>5eee83eaa61eff38f470dc690218bebf73f46e23</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Private.CoreFx.NETCoreApp" Version="4.6.0-preview4.19163.10">
+    <Dependency Name="Microsoft.Private.CoreFx.NETCoreApp" Version="4.6.0-preview5.19224.8">
       <Uri>https://github.com/dotnet/corefx</Uri>
-      <Sha>dc522ef97fac72e64cd74825b7ef497f82af4624</Sha>
+      <Sha>d06ce9d2116cb4bda528822e63d32ce3735ea653</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App" Version="3.0.0-preview4-27518-01">
+    <Dependency Name="Microsoft.NETCore.App" Version="3.0.0-preview5-27626-15">
       <Uri>https://github.com/dotnet/core-setup</Uri>
-      <Sha>ee54d4cbd2b305eadf6f341bdc9d4abccdb50559</Sha>
+      <Sha>61f30f5a23af498908cadd9cc282d4d67d71f522</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.Common.ItemTemplates" Version="1.0.2-beta5.19115.3">
+    <Dependency Name="Microsoft.DotNet.Common.ItemTemplates" Version="1.0.2-beta5.19223.2">
       <Uri>https://github.com/dotnet/templating</Uri>
-      <Sha>0fd9a865045026041f2ea4942eb8c91782193078</Sha>
+      <Sha>e854a58b469c2b4a660fc6e448657d1dd8e40ab9</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.Compilers" Version="3.1.0-beta1-19162-08">
+    <Dependency Name="Microsoft.NETCore.Compilers" Version="3.1.0-beta3-19217-05">
       <Uri>https://github.com/dotnet/roslyn</Uri>
-      <Sha>01f3eb103049e2c93e0516c7d50908031deaca74</Sha>
+      <Sha>79b8271f65609b23e0c0ee1babed6862b7c5ed56</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Build" Version="16.1.0-preview.27">
+    <Dependency Name="Microsoft.Build" Version="16.0.462">
       <Uri>https://github.com/microsoft/msbuild</Uri>
-      <Sha>d004974104fde202e633b3c97e0ece3287aa62f9</Sha>
+      <Sha>62fb89029dac66f73a5ca52363fd11b36f12190b</Sha>
     </Dependency>
-    <Dependency Name="NuGet.Build.Tasks" Version="5.0.0-preview1.5754">
+    <Dependency Name="NuGet.Build.Tasks" Version="5.1.0-rtm.5921">
       <Uri>https://github.com/NuGet/NuGet.Client</Uri>
       <Sha>27af96bdb7ba8d6d7ea9ad53fc76cd1d1aa80703</Sha>
       <RepoName>nuget-client</RepoName>
     </Dependency>
-    <Dependency Name="Microsoft.NET.Sdk" Version="3.0.100-preview4.19165.1">
+    <Dependency Name="Microsoft.NET.Sdk" Version="3.0.100-preview5.19225.2">
       <Uri>https://github.com/dotnet/sdk</Uri>
-      <Sha>814b7898f9908a88f62706331cf56f1ecc9745eb</Sha>
+      <Sha>7af4517bbcf17b000889c1077ef5976f2b6350ea</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.Cli.CommandLine" Version="1.0.0-preview.19127.1">
+    <Dependency Name="Microsoft.DotNet.Cli.CommandLine" Version="1.0.0-preview.19208.1">
```

Comment:
Same here.  The version changes but the Sha doesn't.  What is this Version used for?

Reply:
This has been rebuilt a few times but has no code changes, so the SHA never changed.